### PR TITLE
ACM-3267 add search-v2-operator to must-gather

### PR DESCRIPTION
**Pull request by Anxhela21**
Signed-off-by: Anxhela <acoba@redhat.com>

**Related Issue:**  [JIRA ISSUE](https://issues.redhat.com/browse/ACM-3267)

**Description of Changes:**

**What resource is being added**: search-v2-operator

**Is this a Hub or Managed cluster change?:**
Hub Cluster | Managed Cluster | N/A

**Notes:**

### CODE REVIEW

File: collection-scripts/gather [hunk 1]
```diff
@@ -231,6 +231,8 @@ gather_hub(){
     oc adm inspect  discoveryconfigs.discovery.open-cluster-management.io --all-namespaces  --dest-dir=must-gather
     oc adm inspect searchoperators.search.open-cluster-management.io --all-namespaces  --dest-dir=must-gather
     oc adm inspect searchcustomizations.search.open-cluster-management.io --all-namespaces  --dest-dir=must-gather
```
Comment (jlpadilla) [inline collection-scripts/gather:233]: Let's remove searchoperators and searchcustomizations because those are no longer used.

Comment (Anxhela21) [reply collection-scripts/gather]: Done.

**Comment by Anxhela21** [comment]
verified search-v2-operator resource is now added to must gather:
`must-gather.local.7228924333161258166/quay-io-stolostron-must-gather-sha256-6ab3406a6d15ad6e3ffd1c2a443cf4fa1f04ae5ee5f7c2ee8d60b4654358d208/namespaces/open-cluster-management/pods/search-v2-operator-controller-manager-58475d96d5-t76th/search-v2-operator-controller-manager-58475d96d5-t76th.yaml`

```
apiVersion: v1
kind: Pod
metadata:
  annotations:
    k8s.v1.cni.cncf.io/network-status: |-
      [{
          "name": "openshift-sdn",
          "interface": "eth0",
          "ips": [
              "10.128.1.24"
          ],
          "default": true,
          "dns": {}
      }]
    k8s.v1.cni.cncf.io/networks-status: |-
      [{
          "name": "openshift-sdn",
          "interface": "eth0",
          "ips": [
              "10.128.1.24"
          ],
          "default": true,
          "dns": {}
      }]
    openshift.io/scc: restricted-v2
    seccomp.security.alpha.kubernetes.io/pod: runtime/default
  creationTimestamp: "2023-02-08T16:34:21Z"
  generateName: search-v2-operator-controller-manager-58475d96d5-
  labels:
    control-plane: controller-manager
    ocm-antiaffinity-selector: search-v2-operator-controller-manager
    pod-template-hash: 58475d96d5
  managedFields:
  - apiVersion: v1
    fieldsType: FieldsV1
    fieldsV1:
      f:metadata:
        f:generateName: {}
        f:labels:
          .: {}
          f:control-plane: {}
          f:ocm-antiaffinity-selector: {}
          f:pod-template-hash: {}
        f:ownerReferences:
          .: {}
          k:{"uid":"1689d9d3-b939-43ff-8dad-2e41528ad58f"}: {}
      f:spec:
        f:affinity:
          .: {}
          f:podAntiAffinity:
            .: {}
            f:preferredDuringSchedulingIgnoredDuringExecution: {}
        f:containers:
          k:{"name":"kube-rbac-proxy"}:
            .: {}
            f:args: {}
            f:image: {}
            f:imagePullPolicy: {}
            f:name: {}
            f:ports:
              .: {}
              k:{"containerPort":8443,"protocol":"TCP"}:
                .: {}
                f:containerPort: {}
                f:name: {}
                f:protocol: {}
            f:resources:
              .: {}
              f:limits:
                .: {}
                f:cpu: {}
                f:memory: {}
              f:requests:
                .: {}
                f:cpu: {}
                f:memory: {}
            f:securityContext:
              .: {}
              f:allowPrivilegeEscalation: {}
              f:capabilities:
                .: {}
                f:drop: {}
              f:privileged: {}
              f:readOnlyRootFilesystem: {}
            f:terminationMessagePath: {}
            f:terminationMessagePolicy: {}
          k:{"name":"manager"}:
            .: {}
            f:args: {}
            f:command: {}
            f:env:
              .: {}
              k:{"name":"API_IMAGE"}:
                .: {}
                f:name: {}
                f:value: {}
              k:{"name":"COLLECTOR_IMAGE"}:
                .: {}
                f:name: {}
                f:value: {}
              k:{"name":"INDEXER_IMAGE"}:
                .: {}
                f:name: {}
                f:value: {}
              k:{"name":"POSTGRES_IMAGE"}:
                .: {}
                f:name: {}
                f:value: {}
              k:{"name":"WATCH_NAMESPACE"}:
                .: {}
                f:name: {}
                f:valueFrom:
                  .: {}
                  f:fieldRef: {}
            f:image: {}
            f:imagePullPolicy: {}
            f:livenessProbe:
              .: {}
              f:failureThreshold: {}
              f:httpGet:
                .: {}
                f:path: {}
                f:port: {}
                f:scheme: {}
              f:initialDelaySeconds: {}
              f:periodSeconds: {}
              f:successThreshold: {}
              f:timeoutSeconds: {}
            f:name: {}
            f:readinessProbe:
              .: {}
              f:failureThreshold: {}
              f:httpGet:
                .: {}
                f:path: {}
                f:port: {}
                f:scheme: {}
              f:initialDelaySeconds: {}
              f:periodSeconds: {}
              f:successThreshold: {}
              f:timeoutSeconds: {}
            f:resources:
              .: {}
              f:limits:
                .: {}
                f:cpu: {}
                f:memory: {}
              f:requests:
                .: {}
                f:cpu: {}
                f:memory: {}
            f:securityContext:
              .: {}
              f:allowPrivilegeEscalation: {}
              f:capabilities:
                .: {}
                f:drop: {}
              f:privileged: {}
              f:readOnlyRootFilesystem: {}
            f:terminationMessagePath: {}
            f:terminationMessagePolicy: {}
        f:dnsPolicy: {}
        f:enableServiceLinks: {}
        f:imagePullSecrets:
          .: {}
          k:{"name":"multiclusterhub-operator-pull-secret"}: {}
        f:restartPolicy: {}
        f:schedulerName: {}
        f:securityContext:
          .: {}
          f:runAsNonRoot: {}
        f:serviceAccount: {}
        f:serviceAccountName: {}
        f:terminationGracePeriodSeconds: {}
        f:tolerations: {}
    manager: kube-controller-manager
    operation: Update
    time: "2023-02-08T16:34:21Z"
  - apiVersion: v1
    fieldsType: FieldsV1
    fieldsV1:
      f:metadata:
        f:annotations:
          f:k8s.v1.cni.cncf.io/network-status: {}
          f:k8s.v1.cni.cncf.io/networks-status: {}
    manager: multus
    operation: Update
    subresource: status
    time: "2023-02-08T16:34:23Z"
  - apiVersion: v1
    fieldsType: FieldsV1
    fieldsV1:
      f:status:
        f:conditions:
          k:{"type":"ContainersReady"}:
            .: {}
            f:lastProbeTime: {}
            f:lastTransitionTime: {}
            f:status: {}
            f:type: {}
          k:{"type":"Initialized"}:
            .: {}
            f:lastProbeTime: {}
            f:lastTransitionTime: {}
            f:status: {}
            f:type: {}
          k:{"type":"Ready"}:
            .: {}
            f:lastProbeTime: {}
            f:lastTransitionTime: {}
            f:status: {}
            f:type: {}
        f:containerStatuses: {}
        f:hostIP: {}
        f:phase: {}
        f:podIP: {}
        f:podIPs:
          .: {}
          k:{"ip":"10.128.1.24"}:
            .: {}
            f:ip: {}
        f:startTime: {}
    manager: kubelet
    operation: Update
    subresource: status
    time: "2023-02-08T16:34:41Z"
  name: search-v2-operator-controller-manager-58475d96d5-t76th
  namespace: open-cluster-management
  ownerReferences:
  - apiVersion: apps/v1
    blockOwnerDeletion: true
    controller: true
    kind: ReplicaSet
    name: search-v2-operator-controller-manager-58475d96d5
    uid: 1689d9d3-b939-43ff-8dad-2e41528ad58f
  resourceVersion: "105895"
  uid: 3e0fcab9-10a2-4bba-a3bc-b8bc23120b03
spec:
  affinity:
    podAntiAffinity:
      preferredDuringSchedulingIgnoredDuringExecution:
      - podAffinityTerm:
          labelSelector:
            matchExpressions:
            - key: ocm-antiaffinity-selector
              operator: In
              values:
              - search-v2-operator-controller-manager
          topologyKey: topology.kubernetes.io/zone
        weight: 70
      - podAffinityTerm:
          labelSelector:
            matchExpressions:
            - key: ocm-antiaffinity-selector
              operator: In
              values:
              - search-v2-operator-controller-manager
          topologyKey: kubernetes.io/hostname
        weight: 35
  containers:
  - args:
    - --secure-listen-address=0.0.0.0:8443
    - --upstream=http://127.0.0.1:8080/
    - --logtostderr=true
    - --v=10
    image: quay.io:443/acm-d/kube-rbac-proxy-rhel8@sha256:cdf4253db8d48fbc04e985bc151e8d4c77c03832d69f873663fb2cbdc0e3ecb6
    imagePullPolicy: IfNotPresent
    name: kube-rbac-proxy
    ports:
    - containerPort: 8443
      name: https
      protocol: TCP
    resources:
      limits:
        cpu: 200m
        memory: 1Gi
      requests:
        cpu: 100m
        memory: 256Mi
    securityContext:
      allowPrivilegeEscalation: false
      capabilities:
        drop:
        - ALL
      privileged: false
      readOnlyRootFilesystem: true
      runAsUser: 1000670000
    terminationMessagePath: /dev/termination-log
    terminationMessagePolicy: File
    volumeMounts:
    - mountPath: /var/run/secrets/kubernetes.io/serviceaccount
      name: kube-api-access-7k8n6
      readOnly: true
  - args:
    - --health-probe-bind-address=:8081
    - --metrics-bind-address=127.0.0.1:8080
    - --leader-elect
    command:
    - /manager
    env:
    - name: WATCH_NAMESPACE
      valueFrom:
        fieldRef:
          apiVersion: v1
          fieldPath: metadata.namespace
    - name: POSTGRES_IMAGE
      value: quay.io:443/acm-d/postgresql-13@sha256:826a1a75f4186f36217b5c1f1aa270838c4b55b1021d33a4876566bfc3b8e629
    - name: INDEXER_IMAGE
      value: quay.io:443/acm-d/acm-search-indexer-rhel8@sha256:45223fa492a2189067bd49dcac10b31cb92994ad307a859543d47380d31c80c7
    - name: COLLECTOR_IMAGE
      value: quay.io:443/acm-d/search-collector-rhel8@sha256:a0fc6605f62e4b7670ab96cebeb9fbe8ff4a3aedfb9ad59bf67ae8acaa02aeaa
    - name: API_IMAGE
      value: quay.io:443/acm-d/acm-search-v2-api-rhel8@sha256:99783813e851b536a0f6f0bc81a326e7737a9db4f9fdda0db0ae5c16add98b8f
    image: quay.io:443/acm-d/acm-search-v2-rhel8@sha256:8c487eabbd7a70aefcf71c062c28f2a6de2fa7ad79709bb401150af07b4e3adc
    imagePullPolicy: IfNotPresent
    livenessProbe:
      failureThreshold: 3
      httpGet:
        path: /healthz
        port: 8081
        scheme: HTTP
      initialDelaySeconds: 15
      periodSeconds: 20
      successThreshold: 1
      timeoutSeconds: 1
    name: manager
    readinessProbe:
      failureThreshold: 3
      httpGet:
        path: /readyz
        port: 8081
        scheme: HTTP
      initialDelaySeconds: 5
      periodSeconds: 10
      successThreshold: 1
      timeoutSeconds: 1
    resources:
      limits:
        cpu: 200m
        memory: 1Gi
      requests:
        cpu: 50m
        memory: 256Mi
    securityContext:
      allowPrivilegeEscalation: false
      capabilities:
        drop:
        - ALL
      privileged: false
      readOnlyRootFilesystem: true
      runAsUser: 1000670000
    terminationMessagePath: /dev/termination-log
    terminationMessagePolicy: File
    volumeMounts:
    - mountPath: /var/run/secrets/kubernetes.io/serviceaccount
      name: kube-api-access-7k8n6
      readOnly: true
  dnsPolicy: ClusterFirst
  enableServiceLinks: true
  imagePullSecrets:
  - name: multiclusterhub-operator-pull-secret
  nodeName: ip-10-0-153-148.us-east-2.compute.internal
  preemptionPolicy: PreemptLowerPriority
  priority: 0
  restartPolicy: Always
  schedulerName: default-scheduler
  securityContext:
    fsGroup: 1000670000
    runAsNonRoot: true
    seLinuxOptions:
      level: s0:c26,c10
    seccompProfile:
      type: RuntimeDefault
  serviceAccount: search-v2-operator
  serviceAccountName: search-v2-operator
  terminationGracePeriodSeconds: 10
  tolerations:
  - effect: NoSchedule
    key: node-role.kubernetes.io/infra
    operator: Exists
  - effect: NoExecute
    key: node.kubernetes.io/not-ready
    operator: Exists
    tolerationSeconds: 300
  - effect: NoExecute
    key: node.kubernetes.io/unreachable
    operator: Exists
    tolerationSeconds: 300
  - effect: NoSchedule
    key: node.kubernetes.io/memory-pressure
    operator: Exists
  volumes:
  - name: kube-api-access-7k8n6
    projected:
      defaultMode: 420
      sources:
      - serviceAccountToken:
          expirationSeconds: 3607
          path: token
      - configMap:
          items:
          - key: ca.crt
            path: ca.crt
          name: kube-root-ca.crt
      - downwardAPI:
          items:
          - fieldRef:
              apiVersion: v1
              fieldPath: metadata.namespace
            path: namespace
      - configMap:
          items:
          - key: service-ca.crt
            path: service-ca.crt
          name: openshift-service-ca.crt
status:
  conditions:
  - lastProbeTime: null
    lastTransitionTime: "2023-02-08T16:34:21Z"
    status: "True"
    type: Initialized
  - lastProbeTime: null
    lastTransitionTime: "2023-02-08T16:34:41Z"
    status: "True"
    type: Ready
  - lastProbeTime: null
    lastTransitionTime: "2023-02-08T16:34:41Z"
    status: "True"
    type: ContainersReady
  - lastProbeTime: null
    lastTransitionTime: "2023-02-08T16:34:21Z"
    status: "True"
    type: PodScheduled
  containerStatuses:
  - containerID: cri-o://62e8ef7c5838e67c02c03301e5fac0f78d6c37e375bafc79e9a14ad132049dad
    image: quay.io:443/acm-d/kube-rbac-proxy-rhel8@sha256:cdf4253db8d48fbc04e985bc151e8d4c77c03832d69f873663fb2cbdc0e3ecb6
    imageID: quay.io:443/acm-d/kube-rbac-proxy-rhel8@sha256:95de0275698ef704379246e088763c3827171487049b9cfb857712ddbdef46e9
    lastState: {}
    name: kube-rbac-proxy
    ready: true
    restartCount: 0
    started: true
    state:
      running:
        startedAt: "2023-02-08T16:34:29Z"
  - containerID: cri-o://1b73016c705dcc9f203977918bd2cfec8cc50ebebfe9c95fbb354de02b33c9e2
    image: quay.io:443/acm-d/acm-search-v2-rhel8@sha256:8c487eabbd7a70aefcf71c062c28f2a6de2fa7ad79709bb401150af07b4e3adc
    imageID: quay.io:443/acm-d/acm-search-v2-rhel8@sha256:8c487eabbd7a70aefcf71c062c28f2a6de2fa7ad79709bb401150af07b4e3adc
    lastState: {}
    name: manager
    ready: true
    restartCount: 0
    started: true
    state:
      running:
        startedAt: "2023-02-08T16:34:34Z"
  hostIP: 10.0.153.148
  phase: Running
  podIP: 10.128.1.24
  podIPs:
  - ip: 10.128.1.24
  qosClass: Burstable
  startTime: "2023-02-08T16:34:21Z"
```

**Pull request merged**